.pylintrc

```diff
@@ -2,9 +2,10 @@
 ignore-patterns=serialization.py,message.py,__main__.py,.*_pb2.py
 
 [MESSAGES CONTROL]
-disable=C0103,C0201,C0301,C0302,C0330,W0105,W0107,W1202,W1203,R0902,R0913,R0914,R0801,R0904,R0903,R0911,R0912,R0901,R0916,R1702,R0915
+disable=C0103,C0201,C0301,C0302,C0330,W0105,W0107,W0707,W1202,W1203,R0902,R0913,R0914,R0801,R0904,R0903,R0911,R0912,R0901,R0916,R1702,R0915,R1725
```

Comment:
these were two new checks due to `pylint` version upgrade from `2.5.2` to `2.6.0`